tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -263,6 +263,21 @@ where
     }
 
     /// Sets the layer being built to use a [JSON formatter](../fmt/format/struct.Json.html).
+    ///
+    /// The full format includes fields from all entered spans.
+    ///
+    /// # Example Output
+    ///
+    /// ```ignore,json
+    /// {"timestamp":"Feb 20 11:28:15.096","level":"INFO","target":"mycrate","fields":{"message":"some message", "key": "value"}}
+    /// ```
+    ///
+    /// # Options
+    ///
+    /// - [`LayerBuilder::flatten_event`] can be used to enable flattening event metadata into the root
```

Comment:
I don't think it's actually the metadata that's flattened, but the _fields_ — the metadata is already stored in the root object in the above example.
```suggestion
    /// - [`LayerBuilder::flatten_event`] can be used to enable flattening event fields into the root
```